scripts/dataset_processing/tts/compute_features.py

```diff
@@ -51,9 +52,18 @@ def get_args():
     parser.add_argument(
         "--feature_dir", required=True, type=Path, help="Path to directory where feature data will be stored.",
     )
+    parser.add_argument(
+        "--dedupe_files",
+        action=argparse.BooleanOptionalAction,
+        help="If given, will only process the first manifest entry found for each audio file.",
+    )
+    parser.add_argument(
+        "--overwrite", action=argparse.BooleanOptionalAction, help="Whether to overwrite existing feature files.",
```

Comment:
nit: this is new, not sure if it would be an issue. I usually used `action='store_true'`.

Reply:
Reading some threads online, it sounds like the two are equivalent. `BooleanOptionalAction` was is recommended for Python 3.9+.